### ISSUE TITLE
lecture de l'attribut media_title

### ISSUE THREAD

**Issue by Christ6118** [JackJPowell/hass-psn#26]
**###en Français:** 

Bonjour.
Tout d'abord merci pour votre travail je trouve sa génial.
je tente de l utiliser pour faire dire a mon enceinte connectée bon jeu sur "le nom du jeu"
pour sa j'utilise l'attribut media_title qui qui viens de media_player.gege_6118_ps5_console

j ai tenter plein de chose differente et voici ce que je pense etre bon :

- id: '1733595841808'
  alias: Annoncer le nom du jeu PS5 à l'enceinte Chambre à coucher
  description: ''
  trigger:
    - platform: state
      entity_id: media_player.gege_6118_ps5_console
      attribute: media_title
  condition:
    - condition: template
      value_template: >
        {{ state_attr('media_player.gege_6118_ps5_console', 'media_title') not in ['Browsing the menu', None] }}
    - condition: template
      value_template: >
        {{ trigger.from_state.attributes.media_title != trigger.to_state.attributes.media_title }}
  action:
    - service: tts.google_translate_say
      data:
        entity_id: media_player.chambre_a_coucher
        message: >-
          Bon jeu sur {{ state_attr('media_player.gege_6118_ps5_console', 'media_title') }}
      cache: false
  mode: single

rien ne ce passe ou ai je fais l erreur?

je débute dans home assistant soyer clément :)

**### in english :**

Hello.

First of all, thank you for your work, I find it amazing. I am trying to use it to make my connected speaker say "good game on the name of the game". For this, I use the media_title attribute which comes from media_player.gege_6118_ps5_console.

I have tried many different things, and here’s what I think is good:

- id: '1733595841808'
  alias: Annoncer le nom du jeu PS5 à l'enceinte Chambre à coucher
  description: ''
  trigger:
    - platform: state
      entity_id: media_player.gege_6118_ps5_console
      attribute: media_title
  condition:
    - condition: template
      value_template: >
        {{ state_attr('media_player.gege_6118_ps5_console', 'media_title') not in ['Browsing the menu', None] }}
    - condition: template
      value_template: >
        {{ trigger.from_state.attributes.media_title != trigger.to_state.attributes.media_title }}
  action:
    - service: tts.google_translate_say
      data:
        entity_id: media_player.chambre_a_coucher
        message: >-
          Bon jeu sur {{ state_attr('media_player.gege_6118_ps5_console', 'media_title') }}
      cache: false
  mode: single

Nothing happens, or have I made a mistake?

I’m just starting with Home Assistant, so be gentle :)

**Comment by JackJPowell** [comment]
Hey! Glad you're enjoying the integration. I have a big update that I'm just about to push out that will hopefully simply things for you. In it, there is a configuration option to create all attributes as top level entities. You can get the beta right now by going into the hacs integration and enabling it for the playstation network service.

**Comment by JackJPowell** [comment]
This worked perfectly for me:

```alias: Announce Game
triggers:
  - trigger: state
    entity_id:
      - media_player.jackpowell_ps5_console
    attribute: media_title
conditions:
  - condition: template
    value_template: >
      {{ state_attr('media_player.jackpowell_ps5_console', 'media_title') not in
      ['Browsing the menu', None] }}
  - condition: template
    value_template: >
      {{ trigger.from_state.attributes.media_title !=
      trigger.to_state.attributes.media_title }}
actions:
  - data:
      entity_id: media_player.living_room_speaker
      message: >-
        Have fun playing {{ state_attr('media_player.jackpowell_ps5_console',
        'media_title') }}
    action: tts.google_translate_say
mode: single
```

**Comment by JackJPowell** [comment]
I've released v0.7 which would allow you to interact with entities rather than state attributes. But the automation I posted also works for annoucements. :)
